ee/clickhouse/views/test/test_clickhouse_experiments.py

```diff
@@ -53,7 +44,7 @@ def test_getting_experiments_is_not_nplus1(self) -> None:
             format="json",
         ).json()
 
-        with self.assertNumQueries(FuzzyInt(9, 10)):
+        with self.assertNumQueries(FuzzyInt(8, 9)):
```

Comment:
@neilkakkar I assume there's one less query since we're no longer checking the license?

Reply:
yep makes sense 👍